src/diffbot.js

```diff
@@ -1,79 +1,19 @@
 const fetch = require('node-fetch');
+const axios = require('axios');
+const { response } = require('express');
 
-class Diffbot {
+const sleep = require('util').promisify(setTimeout);
+
+class DiffBot {
```

Comment:
The class has been renamed to Diffbot (lowercase b), as that's how the actual site displays it.

```suggestion
class Diffbot {
```